src/components/CommentForm.js

```diff
@@ -66,14 +64,23 @@ export default class CommentForm extends React.Component {
     const text = this.refs.editor.getContent().replace(/<p>&nbsp;<\/p>$/m, '')
     if (!text || textLength(text) < 2) return false
 
+    const showTagEditor = () => dispatch(showModal('tag-editor', {
+      creating: false,
+      saveParent: this.saveWithTagDescriptions
+    }))
+
     if (newComment) {
       dispatch(createComment(postId, text, this.state.tagDescriptions))
-      .then(({ error }) => {
-        if (error) return
+      .then(action => {
+        if (responseMissingTagDescriptions(action)) return showTagEditor()
+        if (action.error) return
         trackEvent(ADDED_COMMENT, {post: {id: postId}})
       })
     } else {
       dispatch(updateComment(commentId, text, this.state.tagDescriptions))
+      .then(action => {
```

Comment:
`action => responseMissingTagDescriptions(action) && showTagEditor()` is a nice way to one-line this